pkg/embed/manifests/pre-restore/cert-manager.yaml

```
@@ -5,7 +5,7 @@ metadata:
   namespace: kube-system
 spec:
   chart: cert-manager
-  version: v1.7.1
+  version: v1.5.3
```

```suggestion
  version: v1.6.2
```

The current latest version supporting alpha/beta APIs.

---

Locally v.1.5.3 works but v1.6.2 does not.. devenv stalls out with

```
INFO[0000] Creating Kubernetes cluster                   runtime=kind
Creating cluster "dev-environment" ...
 ✓ Ensuring node image (gcr.io/outreach-docker/kindest/node:v1.21.2) 🖼 
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
 ✓ Waiting ≤ 5m0s for control-plane = Ready ⏳ 
 • Ready after 17s 💚
Set kubectl context to "kind-dev-environment"
You can now use your cluster with:

kubectl cluster-info --context kind-dev-environment --kubeconfig /var/folders/r6/5pcbn0696vxc65xhstdwd2v80000gq/T/devenv-kubeconfig-tmp.yaml

Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
INFO[0037] Removing docker image                         image="gcr.io/outreach-docker/giraffe:latest"
gcr.io/outreach-docker/giraffe:latest
INFO[0038] Removing docker image                         image="gcr.io/outreach-docker/outreach-accounts:latest"
gcr.io/outreach-docker/outreach-accounts:latest
INFO[0038] Removing docker image                         image="gcr.io/outreach-docker/templating:latest"
gcr.io/outreach-docker/templating:latest
INFO[0039] Removing docker image                         image="gcr.io/outreach-docker/clientron:latest"
gcr.io/outreach-docker/clientron:latest
INFO[0039] Deploying Manifest                            manifest=01_helmcontroller.jsonnet
INFO[0040] Deploying Manifest                            manifest=02_local-path-provisioner.jsonnet
INFO[0041] Deploying Manifest                            manifest=03_manifests.jsonnet
INFO[0043] Deploying Manifest                            manifest=04_minio.yaml
INFO[0043] Deploying Manifest                            manifest=04_velero.jsonnet
INFO[0044] Deploying Manifest                            manifest=cert-manager.yaml
INFO[0044] Deploying Manifest                            manifest=metrics-server.yaml
INFO[0044] Deploying Manifest                            manifest=nginx-ingress.jsonnet
INFO[0045] Deploying Manifest                            manifest=vault-secrets-operator.jsonnet
INFO[0046] Waiting for pods to be ready                  error="not all pods were ready" pods="[kube-system/helm-install-cert-manager-b9kkz kube-system/helm-install-ingress-nginx-6q47g kube-system/helm-install-metrics-server-kjs6t kube-system/helm-install-minio-x77fk kube-system/helm-install-vault-secrets-operator-btfln kube-system/helm-install-velero-zlgdv local-path-storage/create-pvc-581cd24d-8234-409a-84db-1c8b3603b42e minio/minio-669c5784d-484dc minio/minio-make-bucket-job-p6hzr]"
INFO[0076] Waiting for pods to be ready                  error="not all pods were ready" pods="[kube-system/metrics-server-5cc7ff86d6-868sv local-path-storage/local-path-provisioner-85494db59d-hsw6d]"
INFO[0106] All pods were ready                          
INFO[0107] Fetching application                          app.name=resourcer app.type=bootstrap app.version=v1.5.0
INFO[0109] Deploying application into devenv...          app.name=resourcer app.type=bootstrap app.version=v1.5.0
INFO[0123] Waiting for pods to be ready                  app.name=resourcer app.type=bootstrap app.version=v1.5.0 error="not all pods were ready" pods="[resourcer--bento1a/resourcer-66d6676444-88t7r]"
INFO[0153] All pods were ready                           app.name=resourcer app.type=bootstrap app.version=v1.5.0
INFO[0153] Deploying Manifest                            manifest=snapshot.jsonnet
INFO[0154] All pods were ready                          
INFO[0154] Waiting for snapshot to finish downloading   
INFO[0154] Streaming job logs                           
INFO[0154] Unable to grab pod logs, waiting 5 seconds to try again ...  reason="no pods found for job"
INFO[0159] Unable to grab pod logs, waiting 5 seconds to try again ...  reason="pod status not running, got: Pending"
INFO[0164] Unable to grab pod logs, waiting 5 seconds to try again ...  reason="pod status not running, got: Pending"
INFO[0169] Unable to grab pod logs, waiting 5 seconds to try again ...  reason="pod status not running, got: Pending"
INFO[0174] Unable to grab pod logs, waiting 5 seconds to try again ...  reason="pod status not running, got: Pending"
INFO[0179] Unable to grab pod logs, waiting 5 seconds to try again ...  reason="pod status not running, got: Pending"
INFO[0184] Unable to grab pod logs, waiting 5 seconds to try again ...  reason="pod status not running, got: Pending"
^CERRO[0185] failed to provision from snapshot, destroying intermediate environment  error="failed to setup snapshot infrastructure: snapshot stage job failed: context canceled"
```